setup.py

```diff
@@ -47,10 +48,36 @@
         setup_vals['long_description'] = fh.read()
 
 
+class FailTestException(Exception):
+    """ It provides a failing build """
+    pass
+
+
+class Tests(Command):
+    """ Run test & coverage, save reports as XML """
+
+    user_options = []  # < For Command API compatibility
+
+    def initialize_options(self, ):
+        pass
+
+    def finalize_options(self, ):
+        pass
+
+    def run(self, ):
```

Comment:
These commas are a weird way to declare a function's signature... 😕

Reply:
Ah yeah that was past me's doing. I'll fix here and in my template